src/layouts/Root/components/Footer.tsx

```diff
@@ -85,8 +85,8 @@ const Footer: React.FC = () => (
             padding: 0 ${p => p.theme.space.sm};
           `}
         >
-          Garden is a design system for Zendesk where we grow beautifully simple and accessible UI
-          components.
+          The source of truth for tools, standards, and best practices when building products at
+          Zendesk
```

Comment:
Now we have a repeat between the splash and the footer. The messaging could come off as somewhere between heavy-handed or lacking imagination.

Reply:
@allisonacs @vsri1137 Any thoughts on a direction for this one?

Reply:
We're still known for beautifully simple, so maybe, 

> Garden is the design for Zendesk where we grow beautiful, simple, and accessible UI components. 

We could also get super simplistic and say, 

> Garden is a design system for Zendesk.

@vsri1137 thoughts?

Reply:
Allison, I recall V (or Chels?) suggesting we pivot to more definite phrasing on the latter,

> Garden is the design system for Zendesk.

Reply:
Brand content recommends no beautifully simple and suggested this:

> Garden is the design system for Zendesk where we grow elegant, simple, and accessible UI components.

I might strike _simple_ if we are using _elegant_; I feel _elegance_ implies _simplicity._

Wow @rossmoody I bet you didn't think you were opening a can of worms with this one… but 🐛 here's another nit: We're not just providing components anymore, this site has content guidelines now, and will have design and pattern guidelines soon. Sooooo maybe it's:

> Garden is the design system for Zendesk where we grow elegant and accessible UI components and guidelines.

Whatchoo think @jzempel @rossmoody @vsri1137 ?

Reply:
Oof, my opinion would be to omit this line in the footer altogether. If we want to keep it, the repetition on the homepage doesn't bother me. Could shift it a smidge: `Garden is the design system for Zendesk and source of truth for tools, standards, and best practices when building products.` I'd be down for whatever the group thinks on this one or if @vsri1137 has direction.

Reply:
Unlike the homepage splash, the footer displays on every page. Feels important to retain quality messaging across the site, if possible. We probably do want to mind the length, though, to prevent it from wrapping until the browser drops down to mobile sizes.

Reply:
Do we go with Allison's line for this one? `Garden is the design system for Zendesk where we grow elegant and accessible UI components and guidelines.`

Reply:
Let's hold (or pester) for comment from @vsri1137.

Reply:
OK pester alert received. My two cents. For the footer, I'd keep it short and simple. [Garden is the design system by Zendesk.](https://share.getcloudapp.com/Jru616Ke). There's three concepts to grok in one line, at the bottom of every page. IMHO it doesn't help to call something "elegant." Let the audience just for themselves.

For the main page strapline we could expand: 

**Welcome to Garden**
Garden is the design system for Zendesk. It’s where we grow the components, standards, and tools that product designers use every day.

We could also consider another headline. I didn't spend too much time on these, but if you're interested I could come up with options.

**Rooted in design**
Garden is a design system for Zendesk. It’s where we grow the components, standards, and tools that product designers use every day.

**Right place, right plant**
Garden is a design system for Zendesk. It’s where we grow the components, standards, and tools that product designers use every day.

**Can you dig it?**
Garden is a design system for Zendesk. It’s where we grow the components, standards, and tools that product designers use every day.

And finally we could make the headline super on the nose, to save space:

**Garden is a design system**
It’s where we grow the components, standards, and tools that product designers use to build Zendesk.

Reply:
Also worth mentioning: every good garden needs a can of worms...

Reply:
Niceeeee, I captured these in the "Garden homepage copy" doc we started up with Julia.

Reply:
@vsrixyz can you clarify...

1. Garden is the design system _by_ Zendesk.
1. Garden is the design system _for_ Zendesk.

Reply:
Oh right, or:

1.  Garden is a design system by Zendesk
1.  Garden is the design system by Zendesk
1.  Garden is a design system for Zendesk
1.  Garden is the design system for Zendesk

Reply:
@rossmoody we know we want to pivot to using the definite article – "the design system".

Reply:
Gotcha, only reason I brought it up is cause the screenshot link has `a`.

Reply:
Great Qs. I think "Garden is the design system by Zendesk." is the most accurate. It's *for* everyone.